eliot-service/eliot/app.py

```diff
@@ -132,7 +125,7 @@ def verify_configuration(self):
         This will raise a configuration error if something isn't right.
 
         """
-        for key, opt in self.required_config.options.items():
+        for key, val in get_config_for_class(self.__class__).items():
             self.config(key)
```

Comment:
Eliot doesn't have any namespaced components, so this works fine. If it did have namespaced components, I'm pretty sure this wouldn't work.